process/sp.py

```diff
@@ -34,7 +37,7 @@
 
     # the configuration file should put in the '/configuration' folder located at the same folder as scripts
     # load city-specific configeration file
-    jsonFile = 'configuration/'' + sys.argv[1]
+    jsonFile = 'configuration/' + sys.argv[1]
```

Comment:
can this path building use `os.path.join` too?

Reply:
Done!